src/Middleware/HttpLogging/src/IHttpLoggingInterceptor.cs

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.HttpLogging;
+
+/// <summary>
+/// Callbacks used to extend the HttpLogging middleware.
+/// </summary>
+public interface IHttpLoggingInterceptor
+{
+    /// <summary>
+    /// A callback to customize the logging of the request and response.
```

Comment:
```suggestion
    /// A callback to customize the logging of the request.
```

Reply:
You can customize the response logging from the request callback. E.g. you can enable/disable the response logging flags based on what you see in the request. You can also customize the response body size limit.